scripts/pi-hole/js/groups.js

```diff
@@ -106,7 +166,7 @@ $(function () {
       }
 
       // Reset visibility of ID column
-      data.columns[0].visible = false;
+      data.columns[1].visible = false;
```

Comment:
```suggestion
      data.columns[0].visible = false;
```

ID should still be column `0`, or?

Reply:
Investigating.

Reply:
Yes... It should be `0`.